charts/argocd-apps/values-integration.yaml

```diff
@@ -991,6 +991,21 @@ applications:
           secretKeyRef:
             name: rails-secret-key-base
             key: SECRET_KEY_BASE
+- name: license-finder
+  helmValues:
+    appImage: 
+    replicaCount: 1
+    extraEnv:
+      - name: SECRET_KEY_BASE
+        valueFrom:
+          secretKeyRef:
+            name: rails-secret-key-base
+            key: SECRET_KEY_BASE
+      - name: PUBLISHING_API_BEARER_TOKEN
+        valueFrom:
+          secretKeyRef:
+            name: signon-token-publisher-publishing-api
```

Comment:
Copy-pasto? Should be `signon-token-license-finder-publishing-api` presumably. (similarly in the other file)

Reply:
Ahh right thanks will do